pkgs/applications/version-management/gitlab/gitaly/deps.nix

```diff
@@ -1319,8 +1319,8 @@
     fetch = {
       type = "git";
       url = "https://github.com/ugorji/go";
-      rev = "v1.1.4";
-      sha256 = "0ma2qvn5wqvjidpdz74x832a813qnr1cxbx6n6n125ak9b3wbn5w";
+      rev = "d75b2dcb6bc8";
```

Comment:
Was this generated by the updater script or changed manually? Because running the updater script changes it back.

Reply:
This was generated by the uploader script. 

I don't recall doing this manually.

Reply:
This rev seems to be some commits before `v1.1.2`, so a downgrade. I don't know why `vgo2nix` generated things that way.